### ISSUE TITLE
Is there any guide or documentation on keyword analysis agent?

### ISSUE THREAD

**Issue by W-Park** [fossology/fossology#1395]
### Description

Recently, I upgraded my fossology from 3.3.0 to 3.5.0.
and I found keyword analysis was added at 3.4.0 RC1
(ab30fbf feat(keyword): new-keyword-agent)

So, now I am curious about the feature.
Is there any guidance or documentation that I can refer to?
Simple explanation would do for me.

PS : I suppose what it does is searching for certain 'Keywords'.
       I wonder if I could modify the list of keywords. (which I couldn't find at admin menu)

### Versions

* Last commit id on master:#171d4d(3.5.0)
* Operating System (lsb_release -a):Not relevant

**Comment by ag4ums** [comment]
Hello @W-Park, you can modify the keyword list in this file: `src/copyright/agent/keyword.conf` (as you mentioned) and then can install/reinstall fossology.
if you have already installed version of fossology then you can directly edit the keyword.conf file placed in the below location :  
    if installed from package : `/usr/share/fossology/keyword/agent/keyword.conf`
    if installed from source :  `/usr/local/share/fossology/keyword/agent/keyword.conf`

**Comment by W-Park** [comment]
Hello @ag4ums, thank you so much for such a fast reply.
I was guessing exactly same thing you mentioned. I've got 3.5.0 installed, so I'd better try to modify it at /usr/local/share/fossology/keyword/agent/keyword.conf

Actually, I am pleased to see such function added.
Setting keywords with Web UI and enabling individual account to set the keywords would make it much better, but I am happy with it for now. :)

**Comment by mcjaeger** [comment]
yes, I agree, setting it in the UI, woul dbe even better (maybe in the customize page from the admin section)

**Comment by shaheemazmalmmd** [comment]
May be we have a settings page for keyword, ECC, IPRA agents so that it will be easy for the users to add or edit this keywords related to this agents.
